public/stylesheets/site/2.0/05-region-main.css

```diff
@@ -28,23 +28,19 @@ http://otwcode.github.com/docs/front_end_coding/patterns/supertypes
   padding: 2em 7.5em 3.5em;
 }
 
-#main.error, #main.slow, #main.maintenance {
+#main.errors {
```

Comment:
The `.error` class controls the styling for our flash messages, which have borders, font colors, and shadows. Changing the name we use on our error pages eliminates the need for these styles (besides, we usually use plurals in the `#main` div).